### PR TITLE
Add goto support

### PR DESCRIPTION
**Pull request by kant2002**
As byproduct fix processing `return;` statement.
Add support for '%' operator.

### CODE REVIEW

File: Cesium.CodeGen/Ir/Expressions/BinaryOperators/BinaryOperator.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@ public enum BinaryOperator
     Add, // +
     Subtract, // -
     Multiply, // *
+    Reminder, // %
```
[the remark this answers is not in the document]
Comment (kant2002) [reply Cesium.CodeGen/Ir/Expressions/BinaryOperators/BinaryOperator.cs:8]: This one is funny mistake

[the remark this answers is not in the document]
Comment (ForNeVeR) [reply Cesium.CodeGen/Ir/Expressions/BinaryOperators/BinaryOperator.cs:8]: I am not making fun of this! 😁

[the remark this answers is not in the document]
Comment (kant2002) [reply Cesium.CodeGen/Ir/Expressions/BinaryOperators/BinaryOperator.cs]: Don’t miss opportunity!

**Review by ForNeVeR — APPROVED** [review]

**Comment by ForNeVeR** [comment]
Thanks!

**Pull request merged**